python/lsst/meas/algorithms/sourceSelector.py

```diff
@@ -490,18 +490,55 @@ def apply(self, catalog):
         return selected
 
 
+class RequireFiniteRaDec(pexConfig.Config):
+    """Select sources that have finite RA and Dec sky coordinate values
+
+    This object can be used as a `lsst.pex.config.Config` for configuring
+    the column names to check for "coore_ra" and "coord_dec" keys.
+
+    This will select against objects for which either the RA or Dec coordinate
+    entries are not numpy.isfinite().
+    """
+    raColName = pexConfig.Field(dtype=str, default="coord_ra", doc="Name of column for RA coordinate")
+    decColName = pexConfig.Field(dtype=str, default="coord_dec", doc="Name of column for Dec coordiante")
+
+    def apply(self, catalog):
+        """Apply the sky coordinate requirements to a catalog
+
+        Returns whether the source is selected.
+
+        Parameters
+        ----------
+        catalog : `lsst.afw.table.SourceCatalog` or `pandas.DataFrame`
+                  or `astropy.table.Table`
+            Catalog of sources to which the requirements will be applied.
+
+        Returns
+        -------
+        selected : `numpy.ndarray`
+            Boolean array indicating for each source whether it is selected
+            (True means selected).
+        """
+        selected = (np.isfinite(_getFieldFromCatalog(catalog, self.raColName))
+                    & np.isfinite(_getFieldFromCatalog(catalog, self.decColName)))
+        return selected
+
+
 class ScienceSourceSelectorConfig(pexConfig.Config):
     """Configuration for selecting science sources"""
     doFluxLimit = pexConfig.Field(dtype=bool, default=False, doc="Apply flux limit?")
     doFlags = pexConfig.Field(dtype=bool, default=False, doc="Apply flag limitation?")
     doUnresolved = pexConfig.Field(dtype=bool, default=False, doc="Apply unresolved limitation?")
     doSignalToNoise = pexConfig.Field(dtype=bool, default=False, doc="Apply signal-to-noise limit?")
     doIsolated = pexConfig.Field(dtype=bool, default=False, doc="Apply isolated limitation?")
+    doRequireFiniteRaDec = pexConfig.Field(dtype=bool, default=False, doc="Apply sky coordinates limitation?")
```

Comment:
This doc should be `Apply finite sky coordinate check?` or I guess limitation is okay (since it's how the others are written, sigh) but it needs the `finite`.